docs/services/firecrest.md

```diff
@@ -1,37 +1,86 @@
 [](){#ref-firecrest}
 # FirecREST
 
-FirecREST is a RESTful API for managing High-Performance Computing resources, developed at CSCS.
-Scientific platform developers can integrate Firecrest into web-enabled portals and applications, allowing them to securely access authenticated and authorized CSCS services such as job submission and data transfer on HPC systems.
+FirecREST is a RESTful API for programatically accessing High-Performance Computing resources, developed at CSCS.
+
+Users can make use of FirecREST to automate access to HPC, enabling [CI/CD pipelines](https://github.com/eth-cscs/firecrest/tree/master/examples/CI-pipeline), [Worflow Managers](https://github.com/eth-cscs/firecrest/tree/master/examples/airflow-operators), and other tools agains HPC resources.
+
+Additionally, scientific platform developers can integrate FirecREST into [web-enabled portals](https://firecrest-ui-hpc.cscs.ch) and [applications](https://github.com/eth-cscs/firecrest/tree/master/examples/UI-client-credentials), allowing them to securely access authenticated and authorized CSCS services such as job submission and data transfer on HPC systems.
 
 Users can make HTTP requests to perform the following operations:
 
 * basic system utilities like `ls`, `mkdir`, `mv`, `chmod`, `chown`, among others
 * actions against the Slurm workload manager (submit, query, and cancel jobs of the user)
 * internal (between CSCS systems) and external (to/from CSCS systems) data transfers
 
-For a full feature set, have a look at the latest [FirecREST API specification](https://firecrest-docs.v1.svc.cscs.ch/) deployed at CSCS.
+## FirecREST versions
+
+Starting early 2025, CSCS has introduced a new version of the API: [FirecREST version 2](https://eth-cscs.github.io/firecrest-v2).
+
+Version 2 is faster, easier to use, and more efficient in resource management than its predecesor, therefore, if you are new to FirecREST start directly using **version 2** (if available for your platform).
+
+If you're using **version 1**, we recommend you to port your applications to use the new version.
```

Comment:
Can you provide a timeline for deprecation of v1 to add here?

Reply:
Unfortunately, not yet. We need to see how migration from v1 to v2 develops in the next months. I will come back to this in ~3 months.